### PR TITLE
Scrutinizer Fixes

### PR DESCRIPTION
**Pull request by tomphp**

### CODE REVIEW

File: src/ApplicationConfig.php [hunk 1]
```diff
@@ -54,7 +54,7 @@ public function getIterator()
     }
 
     /**
-     * @return array
+     * @return array<integer|string>
```
Comment (localheinz) [inline src/ApplicationConfig.php]: @tomphp 

Ha, interesting - I think I have seen this before somewhere, wasn't sure whether I should use it or not.

One thing, for consistency, should we use

```php
/**
 * @return array<int|string>
 */
```

instead?

Other than this, we are using the short forms for types.

Comment (localheinz) [inline src/ApplicationConfig.php]: @tomphp 

Ha, interesting - I think I have seen this before somewhere, wasn't sure whether I should use it or not.

One thing, for consistency, should we use

```php
/**
 * @return array<int|string>
 */
```

instead?

Other than this, we are using the short forms for types.

**Comment by localheinz** [comment]
@tomphp 

👍 Other than https://github.com/tomphp/config-service-provider/pull/48#discussion_r78507474, this looks good to me - feel free to merge!

**Comment by tomphp** [comment]
@localheinz tbh I'd not seen the `array<int|string>` format before either, Scrutinzer suggested it. Normally I would've used `int[]|string[]` but that's what it suggested.

**Pull request merged**